### PR TITLE
WEBEU2 - React-Todo Wasiu Idowu 🇳🇬

### PR DESCRIPTION
**Pull request by Hoxtygen**
Initial commit

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,112 @@
...
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      newTask: "",
+      todos: [
```
Comment (maximesalomon) [inline src/App.js:15]: Try to put the data in a data.js file then import it and use setState to add it to your state

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,112 @@
...
+        },
+        {
+          //id: new Date(),
+          id: 6,
+          task: "Solve the mystery surrounding the dancing men",
+          completed: false
+        }
+      ]
+    }
+    this.handleAddTodo = this.handleAddTodo.bind(this);
```
Comment (maximesalomon) [inline src/App.js:55]: if you use arrow functions you won't need to care about bind anymore

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,112 @@
...
+    //this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  
+  
+
+  handleAddTodo = (event) =>{
```
Comment (maximesalomon) [inline src/App.js:62]: you need to make it a variable if you want it to works --> const handleTodo = event => {

Comment (maximesalomon) [inline src/App.js:62]: One there is only 1 parameter you don't need the ()

**Review by maximesalomon — COMMENTED** [review]
Looks really good! Great job Wasiu 🔥

File: src/App.js (hunk 1 above)
Comment (Hoxtygen) [reply src/App.js:15]: Understood.

File: src/App.js (hunk 2 above)
Comment (Hoxtygen) [reply src/App.js:55]: Nice, thanks.

File: src/App.js (hunk 3 above)
Comment (Hoxtygen) [reply src/App.js:62]: Looks like I got carried away and forgot, will be sure to check very well from now on

**Pull request merged**